src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+
+### Use a single name input {: #single-name-input}
+
+Allow your users to enter their name using a single input, unless you have a good reason for 
+separately storing given names, family names, honorifics or other name parts. Using a single name 
+input makes forms less complex, enables cut-and-paste, and makes autofill simpler.
+
+In particular, unless you have good reason not to, don't bother adding a separate input for a 
+prefix/title (like Mrs, Dr or Lord). Users can type that in with their name if they want to. Also, 
+`honorific-prefix` autocomplete currently doesn't work in most browsers, and so adding a field for 
+name prefix/title will break the address form autofill experience for most users.
+
+### Enable name autofill 
+
+Use `name` for a full name:
+
+```html
+<input autocomplete="name" ...>
+```
+
+If you really do have a good reason to split out name parts, make sure to to use appropriate 
+autocomplete values:
+
+* `honorific-prefix`
+* `given-name`
+* `nickname`
+* `additional-name-initial`
+* `additional-name`
+* `family-name`
+* `honorific-suffix`
+
+### Allow international names {: #unicode-matching}
+
+You might want to validate your name inputs, or restrict the characters allowed for name data. However, 
+you need to be as unrestrictive as possible with alphabets. It's rude to be told your name is 'invalid'! 
+
+For validation, avoid using regular expressions that only match Latin characters. Latin-only excludes
+users with names or addresses that include characters that aren't in the Latin alphabet. Allow Unicode 
+letter matching instead—and ensure your back-end supports Unicode securely as both input and output. 
+Unicode in regular expressions is well supported by modern browsers.
+
+{% Compare 'worse' %}
+```html
+<!-- Names with non-Latin characters (such as Françoise or Jörg) are 'invalid'. -->
+<input pattern="[\w \-]+" ...>
+```
+{% endCompare %}
+
+{% Compare 'better' %}
+```html
+<!-- Accepts Unicode letters. -->
+<input pattern="[\p{L} \-]+" ...>
+```
+{% endCompare %}
+
+
+<figure class="w-figure">
+   <video controls autoplay loop muted class="w-screenshot">
+     <source src="https://samdutton.com/unicode-letter-matching.mp4" type="video/mp4">
+   </video>
+  <figcaption class="w-figcaption">Unicode letter matching compared to Latin-only letter matching.</figcaption>
+</figure>
+
+### Allow for a variety of address formats {: #address-variety}
+
+When you're designing an address form, bear in mind the bewildering variety of address formats, 
+even within a single country. Be careful not to make assumptions about 'normal' addresses. (Take a 
+look at [UK Address Oddities!](http://www.paulplowman.com/stuff/uk-address-oddities/) if you're not 
+convinced!)
+
+#### Make address forms flexible {:flexible-address}
+
+Don't force users to [shoe-horn](https://en.wikipedia.org/wiki/Shoehorn#Turn_of_phrase) their address 
```

Comment:
I think we should not distract readers with links to dictionaries (follow our own advice? :) [Thesaurus](https://www.thesaurus.com/browse/shoehorn) is not particularly helpful here but I'd still like us to try rephrasing this.

Reply:
Fair enough :). I've changed.